app/src/views/FieldReportForm/common.ts

```diff
@@ -309,23 +311,23 @@ export const reportSchema: FormSchema = {
         });
 
         // CONTEXT
-        baseSchema = addCondition(
-            baseSchema,
-            value,
-            ['status', 'is_covid_report', 'dtype'],
-            ['summary'],
-            (val): Pick<FormSchemaFields, 'summary'> => {
-                const reportType = getReportType(val?.status, val?.is_covid_report, val?.dtype);
-                if (reportType === 'COVID') {
-                    return {
-                        summary: { forceValue: nullValue },
-                    };
-                }
-                return {
-                    summary: { required: true, requiredValidation: requiredStringCondition },
-                };
-            },
-        );
+        // baseSchema = addCondition(
+        //     baseSchema,
+        //     value,
+        //     ['status', 'is_covid_report', 'dtype'],
+        //     ['summary'],
+        //     (val): Pick<FormSchemaFields, 'summary'> => {
+        //         const reportType = getReportType(val?.status, val?.is_covid_report, val?.dtype);
+        //         if (reportType === 'COVID') {
+        //             return {
+        //                 summary: { forceValue: nullValue },
+        //             };
+        //         }
+        //         return {
+        //             summary: { required: true, requiredValidation: requiredStringCondition },
+        //         };
+        //     },
+        // );
```

Comment:
Please clarify why this code is commented out. If it is no longer needed, consider removing it.

Reply:
Also let's not use inline comments :'(